src/Test/Test.csproj

```diff
@@ -182,25 +74,20 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\libse\LibSE.csproj">
-      <Project>{d6f64cd3-c3ea-4b36-b575-9b3b8a3ca13f}</Project>
-      <Name>LibSE</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\ui\SubtitleEdit.csproj">
-      <Project>{511a5b59-1c35-4719-8536-23b19af9b21a}</Project>
-      <Name>SubtitleEdit</Name>
-    </ProjectReference>
+    <ProjectReference Include="..\libse\LibSE.csproj" />
+    <ProjectReference Include="..\ui\SubtitleEdit.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="MSTest" />
```

Comment:
According to Microsoft, this is the only package required 
https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-getting-started